<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="DESKTOP-HI41LU5" failures="3" tests="7" name="Test1" time="56.713" errors="0" timestamp="2021-12-04T12:11:02 GMT">
  <testcase classname="com.hybridproject.testCases.BaseClass" name="setup" time="6.277"/>
  <testcase classname="com.hybridproject.testCases.TC_LoginTest_001" name="login_Test" time="14.479"/>
  <testcase classname="com.hybridproject.testCases.BaseClass" name="setup" time="1.829"/>
  <testcase classname="com.hybridproject.testCases.TC_LoginDDT_002" name="loginTest" time="16.686"/>
  <testcase classname="com.hybridproject.testCases.TC_LoginDDT_002" name="loginTest" time="3.932">
    <failure type="java.lang.AssertionError" message="expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at com.hybridproject.testCases.TC_LoginDDT_002.loginTest(TC_LoginDDT_002.java:48)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 31 stack frames]]>
    </failure>
  </testcase> <!-- loginTest -->
  <testcase classname="com.hybridproject.testCases.TC_LoginDDT_002" name="loginTest" time="4.341">
    <failure type="java.lang.AssertionError" message="expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at com.hybridproject.testCases.TC_LoginDDT_002.loginTest(TC_LoginDDT_002.java:48)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 31 stack frames]]>
    </failure>
  </testcase> <!-- loginTest -->
  <testcase classname="com.hybridproject.testCases.TC_LoginDDT_002" name="loginTest" time="4.514">
    <failure type="java.lang.AssertionError" message="expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at com.hybridproject.testCases.TC_LoginDDT_002.loginTest(TC_LoginDDT_002.java:48)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 31 stack frames]]>
    </failure>
  </testcase> <!-- loginTest -->
</testsuite> <!-- Test1 -->
